### PR TITLE
Cat Avatar Generator: Fix coding style

### PR DESCRIPTION
**Pull request by fabrixxm**
as per comments in #571

### CODE REVIEW

File: catavatar/catavatar.php [hunk 1]
```diff
@@ -18,48 +18,58 @@
...
 
+	logger('unregistered catavatar');
+}
 
-function catavatar_addon_settings(&$a, &$s) {
-	if(! local_user())
+/**
+ * Cat avatar user settings page
+ */
+function catavatar_addon_settings(&$a, &$s)
```
Comment (MrPetovan) [inline catavatar/catavatar.php]: Obsolete: Please use the following argument list instead: `(\Friendica\App $a, &$s)`

File: catavatar/catavatar.php [hunk 2]
```diff
@@ -18,48 +18,58 @@
...
 	]);
-	return;
 }
 
-function catavatar_addon_settings_post(&$a, &$s) {
-	if(! local_user())
+/**
+ * Cat avatar user settings POST handle
+ */
+function catavatar_addon_settings_post(&$a, &$s)
```
Comment (MrPetovan) [inline catavatar/catavatar.php]: Obsolete: Please use the following argument list instead: `(\Friendica\App $a, &$s)`

File: catavatar/catavatar.php [hunk 3]
```diff
@@ -142,15 +152,15 @@ function catavatar_addon_settings_post(&$a, &$s) {
  * @param $a array
  * @param &$b array
  */
-function catavatar_lookup($a, &$b) {
-	$user = dba::selectFirst('user', ['uid'],['email'=>$b['email']]);
-	
-	$url = $a->get_baseurl().'/catavatar/'.$user['uid'];
+function catavatar_lookup($a, &$b)
```
Comment (MrPetovan) [inline catavatar/catavatar.php]: Obsolete: Please use the following argument list instead: `(\Friendica\App $a, &$b)`

File: catavatar/catavatar.php [hunk 4]
```diff
@@ -168,64 +178,65 @@ function catavatar_module(){}
  *
  * @TODO: support sizes
  */
-function catavatar_content($a) {
-	if ($a->argc < 2 || $a->argc > 3)
+function catavatar_content($a)
```
Comment (MrPetovan) [inline catavatar/catavatar.php]: Obsolete: Please use the following argument list instead: `(\Friendica\App $a)`

File: catavatar/catavatar.php [hunk 5]
```diff
@@ -168,64 +178,65 @@ function catavatar_module(){}
...
 	// Serve from the cache if it is younger than $cachetime
-	if ($cachefile != "" && file_exists($cachefile) && time() - $cachetime < filemtime($cachefile)) {
+	if ($cachefile != "" && file_exists($cachefile) && (time() - $cachetime) < filemtime($cachefile)) {
 		header('Pragma: public');
 		header('Cache-Control: max-age=86400');
 		header('Expires: '. gmdate('D, d M Y H:i:s \G\M\T', time() + 86400));
 		header('Content-Type: image/jpg');
 		readfile($cachefile);
-		exit;
+		killme();
```
Comment (MrPetovan) [inline catavatar/catavatar.php]: Obsolete: `killme()` is deprecated and will be removed in a future version, `exit()` was fine.

File: catavatar/catavatar.php [hunk 6]
```diff
@@ -168,64 +178,65 @@ function catavatar_module(){}
...
+	if ($cachefile != "") {
 		$savedfile = fopen($cachefile, 'w+'); # w+ to be at start of the file, write mode, and attempt to create if not existing.
 		fwrite($savedfile, ob_get_contents());
 		fclose($savedfile);
 		chmod($cachefile, 0755);
 	}
+
 	ob_end_flush();
 
 	killme();
```
Comment (MrPetovan) [inline catavatar/catavatar.php]: Obsolete: `killme()` is deprecated and will be removed in a future version, `exit()` is fine.

File: catavatar/catavatar.php [hunk 7]
```diff
@@ -283,15 +294,15 @@ function build_cat($seed='', $size=0){
...
 		imagealphablending($dest, false);
 		imagesavealpha($dest, true);
 		imagecopyresampled($dest, $cat, 0, 0, 0, 0, $size, $size, CATAVATAR_SIZE, CATAVATAR_SIZE);
 		imagedestroy($cat);
 		$cat = $dest;
 	}
-	
+
 	// restore random seed
 	if($seed) srand();
```
Comment (MrPetovan) [inline catavatar/catavatar.php]: Standards: Please wrap conditionals with braces.

File: catavatar/catavatar.php (hunk 1 above)
Comment (MrPetovan) [reply catavatar/catavatar.php]: Absolutely, I didn't want to suggest a form that wouldn't work out of the box.

**Comment by fabrixxm** [comment]
@tobiasd added :+1: 
@MrPetovan ready for next round!
I know I already asked this: is there a tool to check for code style? With a config already usable?

**Comment by MrPetovan** [comment]
@fabrixxm Yes, check PHP CodeSniffer out, it should use the `ruleset.xml` file at the project root.

**Pull request merged**